badshell is disallowing tildes (~) in the DESTDIR directory to warn people
who use a shell that doesn't expand it to $HOMEDIR.

However, we have such a tilde in our directory (~dfsg) that isn't expanded
to $HOMEDIR nor shouldn't be.

Should be reported upstream.

 -- Faidon Liambotis <paravoid@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -505,7 +505,7 @@ ifneq ($(findstring ~,$(DESTDIR)),)
 	@exit 1
 endif
 
-install: badshell datafiles bininstall $(SUBDIRS_INSTALL)
+install: datafiles bininstall $(SUBDIRS_INSTALL)
 	@if [ -x /usr/sbin/asterisk-post-install ]; then \
 		/usr/sbin/asterisk-post-install $(DESTDIR) . ; \
 	fi
